frontend/app/(defaultLayout)/components/page-frame/header.tsx

```diff
@@ -82,7 +81,7 @@ export function Header({ menu }: React.PropsWithChildren<HeaderProps>) {
                   title="iFixit turns 20"
```

Comment:
I'm pretty sure the title should be removed here, too. And maybe the padding
```suggestion
```

Reply:
Both should go, thanks. 👍

Reply:
https://github.com/iFixit/react-commerce/pull/2195/commits/137aeba2a7166e198ae49964043944e03396d78b